packages/nx-adsp/src/generators/angular-app/angular-app.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  addDependenciesToPackageJson,
+  addProjectConfiguration,
+  formatFiles,
+  generateFiles,
+  getWorkspaceLayout,
+  installPackagesTask,
```

Comment:
return a callback that runs this task to install the dependencies added with addDependenciesToPackageJson

Reply:
This caused some sort of dependency error - that said, I think it worked fine if I manually ran npm install after this was run, but it was kind of confusing. I can add it in.

Reply:
yeah, If I do this,

I get the following when I serve the app

> nx run my-ang-app:serve
Cannot find module 'webpack'
Require stack:
- C:\Users\Jonathan\src\stuff26\node_modules\@angular-devkit\build-webpack\src\webpack\index.js
- C:\Users\Jonathan\src\stuff26\node_modules\@angular-devkit\build-webpack\src\index.js
- C:\Users\Jonathan\src\stuff26\node_modules\@angular-devkit\build-angular\src\dev-server\index.js
- C:\Users\Jonathan\src\stuff26\node_modules\@angular-devkit\architect\node\node-modules-architect-host.js
- C:\Users\Jonathan\src\stuff26\node_modules\@angular-devkit\architect\node\index.js
- C:\Users\Jonathan\src\stuff26\node_modules\@nrwl\tao\src\commands\ngcli-adapter.js
- C:\Users\Jonathan\src\stuff26\node_modules\@nrwl\tao\src\commands\run.js
- C:\Users\Jonathan\src\stuff26\node_modules\@nrwl\tao\index.js
- C:\Users\Jonathan\src\stuff26\node_modules\@nrwl\cli\lib\run-cli.js

and If I install webpack I get

Jonathan@DESKTOP-VQKLAV0 MINGW64 ~/src/stuff26 (master)
$ npm install webpack

added 37 packages, removed 1 package, and audited 1875 packages in 10s

133 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Jonathan@DESKTOP-VQKLAV0 MINGW64 ~/src/stuff26 (master)
$ nx serve my-ang-app

> nx run my-ang-app:serve
The target entry-point "@abgov/angular-components" has missing dependencies:
 - @angular/cdk/overlay


———————————————————————————————————————————————

>  NX   ERROR  Running target "my-ang-app:serve" failed

  Failed tasks:

  - my-ang-app:serve

and when I install 
 npm install @angular/cdk

$ nx serve my-ang-app --verbose

> nx run my-ang-app:serve --verbose
Cannot read property 'add' of undefined

Reply:
seems to be due to --legacy-peer-deps; good to omit the task for now.